db/migrate/20160720045351_add_status_to_pages.rb

```diff
@@ -0,0 +1,19 @@
+class AddStatusToPages < ActiveRecord::Migration
+  class Page < ActiveRecord::Base
+  end
+
+  def up
+    add_column :pages, :publish_status, :integer, default: 1
```

Comment:
I usually don't make use of `null:false` and rely on validations, but I guess it doesn't hurt to add it to help keep the db consistent on its own.